audio-worklet/src/source-controller.js

```diff
@@ -15,48 +15,51 @@
  */
 class SourceController {
   /**
-   * Event Handler and layout for start / stop buttons
+   * Event Handler and layout for start and stop button
    * @param  {String} parentId string id for parent element
-   * @param  {Function} audioStart callback for play button
-   * @param  {Function} audioStop callback for stop button
+   * @param  {Function} startCallback callback for play button
+   * @param  {Function} stopCallback callback for stop button
    */
-  constructor(parentId, audioStart, audioStop) {
-    // Arrange play and stop button and trigger audioStart
-    // and audioStop when these buttons are pressed.
-    parent = document.getElementById(parentId);
-    this.playButton_ = document.createElement('button');
-    this.stopButton_ = document.createElement('button');
-    parent.appendChild(this.playButton_);
-    parent.appendChild(this.stopButton_);
+  constructor(parentId, startCallback, stopCallback) {
+    this.startCallback_ = startCallback;
+    this.stopCallback_ = stopCallback;
 
-    this.playButton_.innerHTML = 'Play';
-    this.stopButton_.innerHTML = 'Stop';
-    this.stopButton_.disabled = true;
+    this.eButtonStartStop_ = document.createElement('button');
```

Comment:
`e` means it holds DOM element reference. Often it becomes `el` or `_El`. This is not from our style, but one of the well-known practices.